packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -120,73 +131,103 @@ class MT5PersonalDetailsForm extends React.Component {
                 >
                     {({ handleSubmit, isSubmitting, errors, touched, values, setFieldValue }) => (
                         <form onSubmit={handleSubmit} autoComplete='off'>
-                            <div className='details-form'>
+                            <Div100vhContainer className='details-form' height_offset='199px' is_disabled={isDesktop()}>
+                                <p className='details-form__description'>
+                                    <Localize
+                                        i18n_default_text={
+                                            'Any information you provide is confidential and will be used for verification purposes only.'
+                                        }
+                                    />
+                                </p>
                                 <div className='details-form__elements-container'>
                                     <ThemedScrollbars autoHide style={{ height: 432 }}>
```

Comment:
@kelly-binary , can you add an `is_native` prop using `isMobile` helper from `@deriv/shared/utils/screen`? 
Because we don't need to use `ThemedScrollbars` on mobile.

Reply:
@zaki-binary sure